apps/condo/domains/organization/integrations/sbbol/sync/MockSbbolResponses.js

```diff
@@ -99,7 +99,6 @@ class MockSbbolResponses {
             ...dvSenderFields,
             name: userInfo.OrgName,
             importId: userInfo.userGuid,
-            importRemoteSystem: SBBOL_IMPORT_NAME,
```

Comment:
why you don't remove importId?

Reply:
I still need some field in object that indicates source user id